torch/csrc/jit/ir/ir.cpp

```diff
@@ -1289,6 +1289,7 @@ Node* Node::replaceWithNewSymbol(Symbol new_symbol) {
     v->replaceAllUsesWith(new_out);
   }
   replace_node->copyMetadata(this);
+  replace_node->copyAttributes(*this);
```

Comment:
Do we need the `copyAttributes` ? An alternative would be to just keep `aten::hardtanh` with the same schema, but we only fuse under the invariants that the min and max is static (which is always true in practice). Then we can do the same thing we currently in the lambda but just by accessing the node inputs

Reply:
I was thinking that this approach would potentially allow one to pass arbitrary values into `MKLDNNHardTanh` and we would now have to enforce the invariant every time we run the op, whereas the attribute approach allows us to fail earlier (at the op construction time) if the invariant is violated. Also, I didn't like the fact that we would be also constantly yanking min and max values on and off the stack and we would need to pass the stack into `aten_op` lambdas which looks like an unnecessary complication(((((

Reply:
The invariant I don't think is a compelling reason (we maintain other invariants in this pass, hardtanh parameters being constant seems like an easy one), but I agree that the moving the min and max values on and off the stack would be unnecessarily complicated. Im good with attribute approach.